package.json

```diff
@@ -34,6 +34,7 @@
     "osd:ciGroup7": "echo apps/data_explorer/aaa_before.spec.js,apps/data_explorer/doc_navigation.spec.js,apps/data_explorer/doc_table.spec.js,apps/data_explorer/errors.spec.js,apps/data_explorer/field_data.spec.js,apps/data_explorer/zzz_after.spec.js",
     "osd:ciGroup8": "echo apps/data_explorer/aaa_before.spec.js,apps/data_explorer/field_visualize.spec.js,apps/data_explorer/filter_editor.spec.js,apps/data_explorer/index_pattern_with_encoded_id.spec.js,apps/data_explorer/index_pattern_without_field.spec.js,apps/data_explorer/zzz_after.spec.js",
     "osd:ciGroup9": "echo apps/data_explorer/aaa_before.spec.js,apps/data_explorer/inspector.spec.js,apps/data_explorer/large_string.spec.js,apps/data_explorer/saved_queries.spec.js,apps/data_explorer/shared_links.spec.js,apps/data_explorer/sidebar.spec.js,apps/data_explorer/source_filter.spec.js,apps/data_explorer/zzz_after.spec.js",
+    "osd:ciGroup10": "echo apps/query_enhancement/*.js",
```

Comment:
what do you mean by this?

Reply:
that's a good call out @ashwin-pc

@abbyhu2000 we start query enhancements with an advanced setting overridden. but doesn't mean everyone will know that.

we should:
* part of a test setup is to set the query enhancements advanced setting to true.
* check if we can re-use some of the MDS test setup. because right now i dont think the tests would be able to much besides things like index patterns as we move the default cluster option and i dont know if the test sets up a call to an external cluster. so you wont be able to select an index. so maybe we can set up on the ci two clusters starting up so that we can add that data soruce as an external data source. this will mean we will need to handle the situation when we start with MDS or not.